app/build.gradle.kts

```
@@ -162,10 +162,13 @@ dependencies {
     implementation(libs.bundlizer.core)
 
     // firebase
-    implementation(platform(libs.firebase.bom))
-    implementation(libs.firebase.fcm)
-    implementation(libs.androidx.work)
-    implementation(libs.googleGms.location)
+    var fdroidBuild = gradle.startParameter.taskRequests.toString().lowercase().contains("fdroid")
+	if (fdroidBuild) {
+        implementation(platform(libs.firebase.bom))
+        implementation(libs.firebase.fcm)
+        implementation(libs.androidx.work)
+        implementation(libs.googleGms.location)
+    }
```

Should this be if not fdroid? and libs.androidx.work is needed even for fdroid

```suggestion
        implementation(libs.androidx.work)
        
    var fdroidBuild = gradle.startParameter.taskRequests.toString().lowercase().contains("fdroid")
	if (!fdroidBuild) {
        implementation(platform(libs.firebase.bom))
        implementation(libs.firebase.fcm)
        implementation(libs.googleGms.location)
    }
```